### ISSUE TITLE
[EN] [RAID] [FIRELANDS] Lord Rhyolith

### ISSUE THREAD

**Issue by stanq87** [Atlantiss/BugTracker#8572]
**Description**: wrong dmg inscrese of erruption debuf from vulcano

**Current behaviour**:  eruption debuf from volcanos increase dmg by 10% per stack

**Expected behaviour**:  eruption debuf from volcanos should increase dmg by 5% per stack

http://wow.gamepedia.com/Lord_Rhyolith

Hotfix (2011-07-14):
    Concussive Stomp damage has been increased on Heroic difficulty.
    Eruption now increases fire damage taken by 5% per stack, down from 10%.
    The Eruption debuff is now cleared when transitioning into phase 2 of the fight."

**Steps to reproduce the problem**: go get stacks

**Revision**: 5254

**Comment by stanq87** [comment]
http://oi63.tinypic.com/2useyb6.jpg

**Comment by Kejmil** [comment]
firelands are prenerf and have  custom buff so... :D

**Comment by Jenovas** [comment]
Not a bug. FL is prenerf and will stay that way.

**Comment by stanq87** [comment]
so "prenerf" +  "custom buff"  , how does it work with all of that 4,2 content and 4.3.4 mecanics ?

**Comment by Jenovas** [comment]
Mechanics are prenerf example Ragnaros uses Magma Geyser, Ryolith debuff is 10%.

With the release of 4.3.4 many spells have been nerfed (talking mainly about Base DMG) that is why all bosses have a custom buff - some 5% some 10% some 15%, the buffs were picked to best match the 4.3.4 nerfs and were farther customized according to the feedback we got from testers.

Players also got buffed - more DMG, more Healing, stronger tanks - we also tried to take that into account when adding the custom buffs. It's not a perfect system, but it's good enough and we pretty much achieved what we wanted (in most of the cases).
